packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -481,6 +481,11 @@ export class AzureCommunicationCallAdapter implements CallAdapter {
       return backendId as UnknownIdentifier;
     });
 
+    if (idsToAdd !== []) {
+      // if we have id's to call then we want to set this state in the adapter to true.
```

Comment:
Question here: I guess idsToAdd includes normal acs id, so we should only set this to true when there is true phoneNumber in it

Reply:
we want to be checking to see if we are calling ACS users as well, both kinds of id's can show in this array